### PR TITLE
fix: First level expand without subquery

### PR DESCRIPTION
**Pull request by jjtang1985**
## Context

Please provide a description of what your change does and why it is needed.

## Definition of Done

Please consider all items and remove only if not applicable.

- [x] Tests created/adjusted for your changes.
- [ ] Release notes updated.
  * Provide sufficient context so that each entry can be understood on its own.
  * Be specific about names of functions, classes, modules, etc.
  * Describe when or where this is relevant
  * Use indicative and present tense. For example, write "Provide function `name` that does X in order to Y" over "Now X can be done by calling a new function".
- [x] PR title adheres to [conventional commit guidelines](https://www.conventionalcommits.org) (please note that only `fix:` and `feat:` will end up in the release notes)
- [ ] If applicable: Properly documented (JSDoc of public API)
- [ ] If applicable: Check if `node run doc` still works.

### CODE REVIEW

File: packages/core/src/odata/common/expandable.ts [hunk 1]
```diff
@@ -13,7 +13,4 @@ export type Expandable<EntityT extends EntityBase> = ODataVersion<
   EntityT
 > extends 'v2'
   ? never
-  :
-      | OneToOneLink<EntityT, any>
-      | OneToManyLink<EntityT, any>
-      | AllFields<EntityT>;
+  : OneToOneLink<EntityT, any> | OneToManyLink<EntityT, any>;
```
Comment (jjtang1985) [inline packages/core/src/odata/common/expandable.ts:16]: I got this from the reference service:
```
Only navigation properties can be expanded.
```

Comment (marikaner) [reply packages/core/src/odata/common/expandable.ts:16]: But you can also expand this: `$expand=*`

File: packages/core/src/odata/v4/uri-conversion/get-expand.ts [hunk 2]
```diff
@@ -37,6 +37,7 @@ function getExpandAsString<EntityT extends Entity>(
   if (expand instanceof Link) {
     params = {
       ...params,
+      // todo why? should delete?
```
Comment (marikaner) [inline packages/core/src/odata/v4/uri-conversion/get-expand.ts:40]: What does this comment mean?

File: test-packages/integration-tests/test/request-builder-v4.spec.ts [hunk 3]
```diff
@@ -56,7 +56,7 @@ describe('Request Builder', () => {
     await expect(request).resolves.not.toThrow();
   });
 
-  it('should resolve when ALL_FIELDS is selected', async () => {
+  it('should resolve when ALL_FIELDS is selected and expand with multiple navigation properties', async () => {
```
Comment (marikaner) [inline test-packages/integration-tests/test/request-builder-v4.spec.ts:59]:
```suggestion
  it('should resolve when ALL_FIELDS is selected and links are expanded', async () => {
```

File: packages/core/src/odata/v4/uri-conversion/get-expand.ts [hunk 4]
```diff
@@ -52,8 +53,9 @@ function getExpandAsString<EntityT extends Entity>(
     const subQuery = Object.entries(params)
       .map(([key, value]) => `${prependDollar(key)}=${value}`)
       .join(',');
+    const subQueryWithBrackets = subQuery ? `(${subQuery})` : '';
 
-    return `${expand._linkedEntity._entityName}(${subQuery})`;
+    return `${expand._fieldName}${subQueryWithBrackets}`;
```
Comment (marikaner) [inline packages/core/src/odata/v4/uri-conversion/get-expand.ts:58]: 👍

**Pull request merged**